src/config.rs

```diff
@@ -118,6 +129,8 @@ impl Default for Config {
 pub struct FarcasterdConfig {
     /// Sets the auto-funding parameters, default to no auto-fund
     pub auto_funding: Option<AutoFundingConfig>,
+    /// Whether checkpoints should be restored manually, or not
+    pub manual_restore: Option<bool>,
```

Comment:
Because the default behavior is restoring checkpoints I believe this option should be `disable_auto_restore` and could find a default value to `false` when generating new config on disk (the `Default` impl on structs).

Reply:
I started out with `disable_auto_restore`, but went with manual restore instead, because I don't like negative options.

Reply:
> Should attempt again without making this argument optional and making it default true?

I didn't understand what you're referring to with "this argument" over here without flipping default from auto- to manual restore.

but perhaps you were saying this: what's your take on `auto_restore` defaulting to `true`?
I think that's unambiguous to understand & document, at the cost of being inconsistent with the other defaults all being `false`.

Reply:
Sorry, this was complete word salad.
Yes, what I was suggesting is adding `auto_restore` and defaulting it to `true`.

Reply:
all good - even King Lear enjoyed word salad from time to time (ok fine: a lot) ;)

> Yes, what I was suggesting is adding auto_restore and defaulting it to true.

let's just go with that then

Reply:
60280a1c9864d4da814748de5984f17e54ee764d